data/json/items/nether.json

```diff
@@ -12,7 +12,7 @@
     "volume": "250 ml",
     "weight": "10 g",
     "symbol": "@",
-    "description": "This defies easy description, it feels solid but the surface is yielding.  Looking at it you have a strange desire to call it a door.",
+    "description": "This defies easy description, it feels solid but the surface is yielding.  Looking at it you have a strange desire to call it a shelter.",
```

Comment:
I don't have an opinion on this change, but this currently seems unrelated to the title of this PR and I don't see an obvious reason for this change. Are you perhaps planning on adding more commits, some of which expect this item to be a shelter of sorts?

Reply:
is is shelter already (it creates 5 minutes long safe space for you) when i was starting this PR i changed it as i was orienting more on Yrax conversation (in alternatives you can read) but it was quickly shut down so i changed what I was doing with this PR but it remained and i belive that to be better description then the same description as rift.